prometheus_flask_exporter/__init__.py

```diff
@@ -82,16 +82,35 @@ def __init__(self, app, path='/metrics',
             (will use the default when `None`)
         :param registry: the Prometheus Registry to use
         """
+        self._options = {
+            'path' : path,
+            'export_defaults' : export_defaults,
+            'group_by_endpoint' : group_by_endpoint,
+            'buckets' : buckets,
+            'registry' : registry,
+        }
+
+        if app:
```

Comment:
@rycus86 This will call `init_app` if the app is passed into the constructor. Is that what you meant?